website/docs/cloud-docs/policy-enforcement/sentinel/import/tfrun.mdx

```diff
@@ -242,7 +243,20 @@ Contains the workspace's [auto-apply](/terraform/cloud-docs/workspaces/settings#
 
 * **Value Type:** Array of strings.
 
-Contains the list of tag names for the workspace.
+Contains the list of tag names for the workspace
+
+### Value: `tag_bindings`
+
+* **Value Type:** Array of objects.
+
+Contains the list of tag bindings, inherited tag bindings, as well as the workspace key-only tags. Each binding has a string `key`, a nullable string `value`, as well as a boolean `inherited` properties.
```

Comment:
Consider: "Contains the complete list of tag bindings for the resource, which also includes inherited tag bindings and the workspace key-only tags."